aks/locals.tf

```diff
@@ -13,8 +13,8 @@ locals {
     node_type_version = "v2"
     node_size         = var.aks_node_sizes.roxie
     single_group      = false
-    min_capacity      = var.aks_capacity.roxie_min
-    max_capacity      = var.aks_capacity.roxie_max
+    min_capacity      = 1
```

Comment:
This should be zero -- or the entire node pool not built at all -- if Roxie is disabled.

Reply:
All min_capacitys are 1 now because when I set them to zero, they still showup on the portal as one.

Reply:
Fixed